### ISSUE TITLE
format_hms can return 2:60

### ISSUE THREAD

**Issue by oskay** [evil-mad/plotink#42]
```
from plotink import text_utils
text_utils.format_hms(179.999)
```
Returns:
'2:60 (Minutes, seconds)'

**Comment by macginitie** [comment]
round(.999) ==> 1

**Comment by macginitie** [comment]
# is this type of thing adequate?
if round(s_elapsed) == 60:
    m_elapsed += 1
    s_elapsed = 0

**Comment by oskay** [comment]
I'll need to think this through and make sure that there aren't additional edge cases (other than 60 minutes) that I've missed.

**Comment by oskay** [comment]
Fix released in 1.3.2, just uploaded to pypi. https://pypi.org/project/plotink/1.3.2/
